visualization/app/codeCharta/ui/mapTreeView/mapTreeView.component.ts

```diff
@@ -33,7 +33,7 @@ export class MapTreeViewController implements CodeMapPreRenderServiceSubscriber,
 		const compareFunction: CompareFunction =
 			sortingOption === SortingOption.NUMBER_OF_FILES
 				? (a, b) => b.attributes[NodeMetricDataService.UNARY_METRIC] - a.attributes[NodeMetricDataService.UNARY_METRIC]
-				: (a, b) => (b.name > a.name ? -1 : 1)
+				: (a, b) => a.name.localeCompare(b.name)
```

Comment:
```suggestion
				: (a, b) => a.name.localeCompare(b.name, undefined, { numeric: true, sensitivity: 'base' })
```

Ideally we add a default collator while starting the application (`new Intl.Collator([...])`).